### PR TITLE
OE-6445 - Ability to tag medications/drugs with attributes (e.g., Type:Anti-coagulant, High-Risk, PreservativeFree, etc)

### PR DESCRIPTION
**Pull request by edgaaar8**

### CODE REVIEW

**Comment by edgaaar8** [comment]
Mike, can you review this before further functional tests, please? Many thanks

**Comment by splatEric** [comment]
The Tag model seems to be in the wrong place to me - if it's being attached to core models, it should be a part of core, not buried in the prescription module.

File: protected/controllers/oeadmin/FormularyDrugsController.php [hunk 1]
```diff
@@ -91,7 +91,7 @@ public function actionList()
      */
     public function actionEdit($id = false)
     {
-        Yii::import('application.modules.OphDrPrescription.models.Tag');
+        Yii::import('application.models.Tag');
```
Comment (splatEric) [inline protected/controllers/oeadmin/FormularyDrugsController.php]: is this import even necessary ... I don't think I've used the Yii::import over a use statement ... what are the benefits?

Comment (neilsf) [inline protected/controllers/oeadmin/FormularyDrugsController.php]: That's right, it's no longer necessary.

**Pull request merged**